### DISCUSSION TITLE
봇 문의드려요..

### DISCUSSION THREAD

**Discussion by papagogo1** [Lastorder-DC/chatreader-kor#52] [문의] [answered]
계정을 처서 연결했는데, 계속 초기화 중에서 안넘어가네요..

중복된게 있는지 없는지 알기위해서 밑에 질문,답글도 봤는데..

쿠키삭제를 해도 마찬가지였습니다..

해결 방법 없을까요..?

**Comment by Lastorder-DC** [comment, marked as answer]
19b4c4f943e515971e6a615dce5d0e765edf99ce 사용 불가능한 TAPIC 대신 tmi.js로 교체되었습니다.
